addons/event/controllers/main.py

```diff
@@ -23,3 +26,55 @@ def event_ics_file(self, event, **kwargs):
             ('Content-Length', len(content)),
             ('Content-Disposition', content_disposition('%s.ics' % event.name))
         ])
+
+    @route(['''/event/tickets_dl'''], type='http', auth="public")
+    def event_tickets_dl(self, dl_hash, event_id, registration_ids, badge_mode=False):
+        """ Returns a pdf response, containing all tickets for attendees in registration_ids for event_id.
+        They can be filtered out based on conditions in `_fetch_registrations_for_event_tickets_dl`
+        Redirects to the portal if not registration is valid / hash is invalid / parameters are missing.
+        This route is used in links in emails to attendees, as well as in registration confirmation screens.
+
+        :param badge_mode: boolean, True to use template of foldable badge instead of full page ticket.
```

Comment:
Would be better if we respect the order of the params and put this one after the others 🧦